jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -547,10 +549,22 @@ public ColumnSortList reloadColumnSortList() {
                 if (optional.isPresent()) {
                     Column col = optional.get().getColumn();
                     col.setDataStoreName(gridSortedColumnPreference.getDataStoreName());
-                    columnSortList.push(new ColumnSortList.ColumnSortInfo(col, gridSortedColumnPreference.isAscending()));
+                    defferredColumnSortListPush(columnSortList,new ColumnSortList.ColumnSortInfo(col, gridSortedColumnPreference.isAscending()));
                 }
             }
         }
         return columnSortList;
     }
+
+    @Override
+    public ColumnSortList.ColumnSortInfo defferredColumnSortListPush(ColumnSortList columnSortList, ColumnSortList.ColumnSortInfo columnSortInfo) {
```

Comment:
perhpaps just leave it as protected method and dont add to interface?

Reply:
I will use the method on unit test,
doAnswer(answer -> {
            Column<T, String> column = new Column<T, String>(new TextCell()) {
                @Override
                public String getValue(T domain) {
                    return null;
                }
            };
            column.setSortable(true);
            column.setDataStoreName("test");
            columnSortList.push(column);
            return columnSortList.get(0);
        }).when(getView()).defferredColumnSortListPush(any(),any());
So I need add it into interface.

Reply:
no really, test uses AbstractMultiGridView type, and even protected works fine.